README.md

```diff
@@ -1,18 +1,152 @@
-# Deploy HPCC Systems Platform with Terraform
-
-This set of Terraform examples deploys all the available features that come with the HPCC Systems OSS Terraform modules.
-
-## Order of deployment
-| Order | Name      | Required |
-| ----- | --------- | :------: |
-| 1     | `VNet`    |   yes    |
-| 2     | `AKS`     |   yes    |
-| 3     | `Storage` |    no    |
-| 4     | `Logging` |    no    |
-| 5     | `AKS`     |    no    |
-| 6     | `HPCC`    |   yes    |
-
-## Modules
-|                Name                 | Source | Used in |
-| :---------------------------------: | :----: | :-----: |
-| `terraform-azurerm-virtual-network` |        | `VNet`  |
\ No newline at end of file
+# Deploy HPCC Systems on Azure under Kubernetes
+
+NOTE: A tutorial of this Terraform for the developer, or others who are interested, can be found [here](documentation/hpcc-tf-for-developers.md).
+
+This is a slightly-opinionated Terraform module for deploying an HPCC Systems cluster on Azure's kubernetes service (aks).  The goal is to provide a simple method for deploying a cluster from scratch, with only the most important options to consider.
+
+The HPCC Systems cluster created by this module uses ephemeral storage, which is the default. This means the storage will be deleted when the cluster is deleted) But, you can also have Persistent Storage.  See the section titled [Persistent Storage](#persistent-storage), below.
+
+This repo is a fork of the excellent work performed by Godson Fortil.  The original can be found in branch, HPCC-27615 of [https://github.com/gfortil/Terraform-azurerm-hpcc].
```

Comment:
Please delete this paragraph.

Reply:
fixed. I deleted the paragraph, "This repo is a fork of the excellent work performed by Godson Fortil. ....f